code/src/main/java/com/google/sps/data/NegativeUserLocations.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.data;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import java.time.Instant;
+import java.util.ArrayList;
+
+@Entity
+public class NegativeUserLocations {
```

Comment:
It now occurs to me that we might have someone use this feature multiple times. How would we need to adjust this class to do that/

Reply:
Not sure I understand

Reply:
If a user logs in as a "negative user" once, and then does so again a week later, will we update thier data? What if they log in after 2+ weeks? How does the negative-user-journey change if they opt to use the service multiple times?

Reply:
I think this is an outdated comment since I changed the data type and  this one is no longer implemented